vector/src/main/java/im/vector/riotx/features/home/room/detail/composer/ComposerEditText.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package im.vector.riotx.features.home.room.detail.composer
+
+import android.content.Context
+import android.net.Uri
+import android.os.Build
+import android.util.AttributeSet
+import android.view.inputmethod.EditorInfo
+import android.view.inputmethod.InputConnection
+import android.widget.EditText
+import androidx.core.view.inputmethod.EditorInfoCompat
+import androidx.core.view.inputmethod.InputConnectionCompat
+
+class ComposerEditText @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = android.R.attr.editTextStyle)
+    : EditText(context, attrs, defStyleAttr) {
+
+    interface Callback {
```

Comment:
Replace by `interface Callback: ComposerEditText.Callback` as the two interfaces are identical?

Reply:
It's just the first step, will get more functions later (the ones handled in RoomDetail at the moment)

Reply:
and? Interface inheritance does not prevent from adding more functions later on

Reply:
Ah yes, didn't understand what you said :O